go.mod

```diff
@@ -1,4 +1,4 @@
-module astuart.co/go-robinhood
+module github.com/chennakrishna8/go-robinhood
```

Comment:
The module base should stay `astuart.co`, but I get why you have changed it. One downside of go modules' current implementation. 😊

Reply:
yep. I did not see that. I will fix it.

Reply:
fixed it.